apps/token/src/routes/governance/test-helpers/mocks.ts

```diff
@@ -0,0 +1,57 @@
+import { NETWORK_PARAMS_QUERY } from '@vegaprotocol/web3';
+import type { VegaKeyExtended } from '@vegaprotocol/wallet';
+import type { MockedResponse } from '@apollo/client/testing';
+import type { NetworkParamsQuery } from '@vegaprotocol/web3';
+
+export const mockPubkey = '0x123';
+const mockKeypair = {
+  pub: mockPubkey,
+} as VegaKeyExtended;
+
+export const mockWalletContext = {
+  keypair: mockKeypair,
+  keypairs: [mockKeypair],
+  sendTx: jest.fn().mockReturnValue(Promise.resolve(null)),
+  connect: jest.fn(),
+  disconnect: jest.fn(),
+  selectPublicKey: jest.fn(),
+  connector: null,
+};
+
+const mockEthereumConfig = {
+  network_id: '3',
+  chain_id: '3',
+  confirmations: 3,
+  collateral_bridge_contract: {
+    address: 'bridge address',
+  },
+};
+
+export const networkParamsQueryMock: MockedResponse<NetworkParamsQuery> = {
+  request: {
+    query: NETWORK_PARAMS_QUERY,
+  },
+  result: {
+    data: {
+      networkParameters: [
+        {
+          __typename: 'NetworkParameter',
+          key: 'blockchains.ethereumConfig',
+          value: JSON.stringify(mockEthereumConfig),
+        },
+      ],
+    },
+  },
+};
+
+export const amendToDate = (amendmentInSeconds: number) => {
+  const now = new Date();
+  return new Date(now.getTime() + amendmentInSeconds * 1000);
+};
+
+const oneWeek = 60 * 60 * 24 * 7;
+
+export const lastMonth = amendToDate(-(oneWeek * 4));
```

Comment:
chore: not the biggest fan of this. I think I'd prefer to set the system time and use static data as this could be flakey. We do this elsewhere.